rust/hyper/Cargo.toml

```
@@ -0,0 +1,22 @@
+[package]
+name = "server"
+version = "0.0.0"
+edition = "2021"
+
+[dependencies]
+http = "1.2.0"
+http-body-util = "0.1.2"
+hyper = { version = "1.5.2", features = ["full"] }
```

```suggestion
hyper = { version = "1.5", features = ["full"] }
```

what's the default valud of _features_ ?